config/modules.config

```diff
@@ -55,51 +55,64 @@ process {
     withName: BUSCO_EUKCC_QC {
         publishDir = [
             [
-                path: "${params.outdir}/qc/",
+                path: "${params.outdir}/qc/eukaryotes/",
                 mode: params.publish_dir_mode,
                 failOnError: true,
             ]
         ]
     }
 
-    withName: CAT {
+    withName: "CHECKM2" {
         publishDir = [
             [
-                path: "${params.outdir}/intermediate_steps/cleaning_bins/",
+                path: "${params.outdir}/qc/prokaryotes/checkm2",
                 mode: params.publish_dir_mode,
-                failOnError: false,
+                pattern: "*.?sv"
             ]
         ]
     }
 
-    withName: "CHECKM2" {
+    withName: "CHECKM2_REFINE" {
```

Comment:
I recently learned that the withName selector: 

```
withName selector applies to a process even when it is included from a module under an alias. For example, withName: hello will apply to any process originally defined as hello, regardless of whether it is included under an alias. Similarly, it will not apply to any process not originally defined as hello, even if it is included under the alias hello
```

so we can merge all the publishDirs into a single one (I think):
```groovy
withName: "CHECKM2" {
    publishDir = [
        [
            path: "${params.outdir}/qc/prokaryotes/checkm2",
            mode: params.publish_dir_mode,
            pattern: "*.?sv"
        ],
        [
            path: "${params.outdir}/intermediate_steps/prokaryotes/checkm2/refinement",
            mode: params.publish_dir_mode,
            pattern: "all_stats.csv"
        ],
        [
            path: "${params.outdir}/intermediate_steps/prokaryotes/refinement/checkm2",
            mode: params.publish_dir_mode,
            pattern: "binner1_filtered_genomes.tsv"
        ],
        [
            path: "${params.outdir}/intermediate_steps/prokaryotes/refinement/checkm2",
            mode: params.publish_dir_mode,
            pattern: "binner2_filtered_genomes.tsv"
        ],
        [
            path: "${params.outdir}/intermediate_steps/prokaryotes/refinement/checkm2",
            mode: params.publish_dir_mode,
            pattern: "binner3_filtered_genomes.tsv"
        ]
    ]
}
```
the problem is that the first publishDir one will capture all the files